### PR TITLE
Feature/#8 Create styled progress bar

### PR DESCRIPTION
**Pull request by MangriMen**

### CODE REVIEW

**Comment by FFreestyler** [comment]
Why progress bar not under avatar box?
image

**Review by FFreestyler — APPROVED** [review]

**Review by StepanovRoma — APPROVED** [review]

File: src/components/charlist/LiquidBar.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        position="absolute"
+        top="0"
+        bgcolor={bgcolor_}
+        width={`${filledBarWidth}%`}
+      />
+      <BarLiquidBlock
+        bgcolor={bgcolor_}
+        position="absolute"
+        height="333%"
+        left={`calc(${waveOffset}% - (${height_} * 3.33) - ${
```
Comment (IHiFiveI) [inline src/components/charlist/LiquidBar.tsx]: Move to helper and make human-readable

File: src/components/charlist/LiquidBar.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  return (
+    <Box
+      position="relative"
+      height={height_}
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      boxSizing="border-box"
+      overflow="hidden"
+      {...props}
```
Comment (IHiFiveI) [inline src/components/charlist/LiquidBar.tsx:118]: As far as I can see, theese rest props right now are used only for styles, so may be make it take MUI `sx` instead?

**Comment by MangriMen** [comment]
[Create styled progress bar](https://trello.com/c/yqDJFwqw/8-create-styled-progress-bar)

**Comment by MangriMen** [comment]
image

**Review by IHiFiveI — APPROVED** [review]

**Pull request merged**